app/server/lib/DocSnapshots.ts

```diff
@@ -375,7 +386,8 @@ export function shouldKeepSnapshots(snapshots: ObjSnapshotWithMetadata[], snapsh
       return false;
     }
 
-    let keep = index < 5;   // Keep 5 most recent versions
+    let keep = index < ( process.env.NUM_SNAPSHOT_KEEP || 5);   // Keep 5 most recent versions if NUM_SNAPSHOT_KEEP not exist
```

Comment:
What do you think about an environment variable that goes well with `GRIST_SNAPSHOT_TIMEBUCKET_CAP`, like `GRIST_SNAPSHOT_RECENT_CAP`? Also maybe `TIMEBUCKET` could be simplified to `TIME`.
The type of `process.env.NUM_SNAPSHOT_KEEP` is a string (or undefined), so it needs some casting (perhaps with `optIntegerParam`, available in same file as `integerParam`)

Reply:
Could you please provide a more detailed description?

Reply:
I assume you mention to rename GRIST_SNAPSHOT_TIMEBUCKET_CAP to GRIST_SNAPSHOT_TIME_CAP. I have now completed all and will push the modifications.

Reply:
I assume you mentioned to rename GRIST_SNAPSHOT_TIMEBUCKET_CAP to GRIST_SNAPSHOT_TIME_CAP. I have now completed the modifications as you have suggested and will push for review.

Reply:
I was trying to suggest renaming `NUM_SNAPSHOT_KEEP` to be more like the other environment variable you're adding, just for some consistency.